configurations/create_kenya_pool.py

```diff
@@ -0,0 +1,156 @@
+import json
+
+from core_data_modules.cleaners import swahili
+from core_data_modules.data_models import CodeScheme
+
+from src.common.configuration import RapidProClientConfiguration, CodaClientConfiguration, UUIDTableClientConfiguration, \
+    EngagementDatabaseClientConfiguration
+from src.engagement_db_to_coda.configuration import CodaSyncConfiguration, CodaDatasetConfiguration, \
+    CodeSchemeConfiguration
+from src.engagement_db_to_rapid_pro.configuration import EngagementDBToRapidProConfiguration, DatasetConfiguration, \
+    WriteModes, ContactField
+from src.pipeline_configuration_spec import PipelineConfiguration, RapidProSource, CodaConfiguration, RapidProTarget
+from src.rapid_pro_to_engagement_db.configuration import FlowResultConfiguration
+
+
+def load_code_scheme(fname):
+    with open(f"code_schemes/{fname}.json") as f:
+        return CodeScheme.from_firebase_map(json.load(f))
+
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="Create-Kenya-Pool",
+    description="Creates the initial Kenya Pool from demographics responses to COVID19, COVID19-Ke-Urban, "
+                "UNDP-Kenya, UNICEF-KENYA, OXFAM-KENYA, and World Vision.",
```

Comment:
```suggestion
                "UNDP-Kenya, UNICEF-KENYA, OXFAM-KENYA, World Vision and GPSDD",
```

Reply:
converted to a todo to add in a future pr